tests/integration_tests/substrate/vanchorTests.spec.ts

```diff
@@ -0,0 +1,365 @@
+import { JsNote, JsNoteBuilder, OutputUtxoConfig } from '@webb-tools/wasm-utils/njs/wasm-utils-njs.js';
+import { ApiPromise } from '@polkadot/api';
+import { decodeAddress, Keyring } from '@polkadot/keyring';
+import { KeyringPair } from '@polkadot/keyring/types';
+import { currencyToUnitI128, KillTask, preparePolkadotApi, startWebbNode, transferBalance } from '../../utils/index.js';
+import { ProvingManagerSetupInput, ProvingManagerWrapper } from '@webb-tools/sdk-core/index.js';
+import { hexToU8a, u8aToHex } from '@polkadot/util';
+import { polkadotTx } from '@webb-tools/test-utils/index.js';
+import path from 'path';
+import fs from 'fs';
+
+let apiPromise: ApiPromise | null = null;
+let keyring: {
+  bob: KeyringPair;
+  alice: KeyringPair;
+  charlie: KeyringPair;
+} | null = null;
+let nodes: KillTask | undefined;
+
+const BOBPhrase = 'asthma early danger glue satisfy spatial decade wing organ bean census announce';
+
+function getKeyring() {
+  if (keyring) {
+    return keyring;
+  }
+  const k = new Keyring({ type: 'sr25519' });
+  const bob = k.addFromMnemonic(BOBPhrase);
+  const alice = k.addFromUri('//Alice');
+  const charlie = k.addFromUri('//Charlie');
+  keyring = {
+    bob,
+    alice,
+    charlie
+  };
+  return keyring;
+}
+
+function generateVAnchorNote(amount: number, chainId: number, outputChainId: number, index?: number) {
+  const noteBuilder = new JsNoteBuilder();
```

Comment:
What's the reason we aren't using sdk-core here again?

Reply:
I will change it to `sdk-core` for consistency

Reply:
Sounds good, would also be great to change the other tests if we are using wasm-utils directly.